api/src/opentrons/protocol_api/protocol_context.py

```diff
@@ -558,9 +559,9 @@ def load_instrument(
                                     the version.
         :param mount: The mount in which this instrument should be attached.
                       This can either be an instance of the enum type
-                      :py:class:`.types.Mount` or one of the strings `'left'`
+                      :py:class:`.Mount` or one of the strings `'left'`
```

Comment:
Looks like an accidental search+replace change.

```suggestion
                      :py:class:`.types.Mount` or one of the strings `'left'`
```

Reply:
It was intentional, but only because I misunderstood what was required for sphinx to understand these. Turns out either works, it just displays differently. Sphinx is weird.

Also, looks like #11379 broke the docs... so that's cool. Will revert docstring changes in this PR and leave docs fixups to another PR